### DISCUSSION TITLE
wrong path used for opening files when using gvfs-smb

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2592] [Q&A] [answered]
Hello!
When opening an smb share in pcmanfm-qt and double-clicking a file, the path smb://example.org will be used for starting the application which will fail. It must use the gvfs path in /run.
How can I further diagnose that?

kind regards

**Comment by tsujan** [comment]
`run` paths are given to apps. Have you installed the required GVFS packages, especially `gvfs-smb`?

**Reply by tsujan** [reply]
I don't have SMB here, but it makes no difference to pcmanfm-qt (lifm-qt) whether it's SMB or SFTP — these things are handled by GLib and GVFS. In the following screenshot, I've opend `sftp://pedram@192.168.1.3/home/pedram/ftp.txt` with FeatherPad, but FeatherPad has received `/run/user/1000/gvfs/sftp:host=192.168.1.3,user=pedram/home/pedram/ftp.txt`:

![SFTP](https://github.com/user-attachments/assets/fbe70a15-f061-4428-9aaa-ea791b60ce8e)

I intentionally used FeatherPad as an example because I've written its code and know that it can't handle SFTP by itself.

Anyway, the problem can't be in pcmanfm-qt.

**Comment by ghost** [comment]
Am 26.07.2024 um 05:44:47 Uhr schrieb tsujan:

> `run` paths are given to apps. Have you installed the required GVFS
> packages, especially `gvfs-smb`?

Yes, I've installed that.

**Comment by ghost** [comment, marked as answer]
Am 26.07.2024 um 06:03:47 Uhr schrieb tsujan:

> Anyway, the problem can't be in pcmanfm-qt.

The problem was that the package gvfs-fuse was missing. gvfs-smb etc.
don't depend on it on Fedora.

**Reply by tsujan** [reply]
I wanted to mention the fuse daemon but, after seeing that it belonged to the `gvfs` package in Arch, I thought you should already have it.

It's always good to mention the distro when asking for help. Anyway, good to know that your problem is solved.
